grails-app/services/org/pih/warehouse/inventory/InventoryService.groovy

```diff
@@ -3425,4 +3425,20 @@ class InventoryService implements ApplicationContextAware {
             }
         }
     }
+
+    void recallInventoryItem(InventoryItem inventoryItem) {
```

Comment:
Given how massive the InventoryService class is we should be thinking longer term about how to refactor this into smaller classes. 

I think the obvious options are 
* Domain (Product, Inventory, InventoryItem) which we used and caused this mess
* Functional area (transfer vs movement vs recall) which seems way better but might have some overlap that we need to contend with 
* Other suggestions?